src/components/Button/Button.jsx

```diff
@@ -21,7 +21,13 @@ const propTypes = {
       PropTypes.arrayOf(PropTypes.node),
    ]).isRequired,
    /** Choose a button design. */
-   design: PropTypes.oneOf(['default', 'primary', 'outline', 'plain']),
+   design: PropTypes.oneOf([
+      'default',
+      'primary',
+      'secondary',
+      'outline',
+      'plain',
+   ]),
```

Comment:
Maybe pull this list out into a const?

Reply:
I think it's alright in this context to keep it how it is. We don't use this list anywhere else, so it won't ever need to be duplicated.